source/features/bugs-tab.tsx

```diff
@@ -111,7 +119,7 @@ async function init(): Promise<void | false> {
 	// Disable unwanted behavior #3001
 	bugsTab.removeAttribute('data-hotkey');
 	bugsTab.removeAttribute('data-selected-links');
-	issuesTab.removeAttribute('data-selected-links');
+	bugsTab.removeAttribute('id');
```

Comment:
Ensure that this bug isn't reintroduced: https://github.com/sindresorhus/refined-github/issues/3001

Reply:
Confirmed it's fine at this moment 👍